### PR TITLE
Delete standup

### PR DESCRIPTION
**Pull request by SyedSibtainRazvi**
# Related Issue

- [169](https://github.com/code-mancers/gather-bot/issues/169)

# functionalities Added or Changed
- added the endpoint for deleting the stand-up

# Additional Info

- In the g-bot dashboard, the admin should be able to see the delete feature along with edit and update
The admin should be able to delete the all existing standup created by the admin

# Checklist

- [ ] Tests
- [ ] Uploaded screenshots

# Screenshots

### CODE REVIEW

File: server/src/services/standups.service.ts [hunk 1]
```diff
@@ -157,6 +157,20 @@ export const getAllStandups = async () => {
...
 
+export const deleteStandups = async (id: string) => {
+  try {
+    const standup = await prisma.standup.delete({
+      where: {
+        id: Number(id),
+      },
+      include: { questions: true, members: true, channel: true },
+    });
+    console.log(standup);
```
Comment (SujayPrabhu96) [inline server/src/services/standups.service.ts:168]: remove

File: server/src/services/standups.service.ts [hunk 2]
```diff
@@ -157,6 +157,20 @@ export const getAllStandups = async () => {
   }
 };
 
+export const deleteStandups = async (id: string) => {
+  try {
+    const standup = await prisma.standup.delete({
+      where: {
+        id: Number(id),
+      },
+      include: { questions: true, members: true, channel: true },
```
Comment (SujayPrabhu96) [inline server/src/services/standups.service.ts:166]: If we are deleting all records associated with the standup, do we need this `include` ?

File: server/src/services/standups.service.ts (hunk 1 above)
Comment (SyedSibtainRazvi) [reply server/src/services/standups.service.ts:168]: Removed the log

File: server/src/services/standups.service.ts (hunk 2 above)
Comment (SyedSibtainRazvi) [reply server/src/services/standups.service.ts:166]: I used include to define which relations to include for the return value while figuring it out, can be removed

File: server/src/services/standups.service.ts [hunk 3]
```diff
@@ -157,6 +157,18 @@ export const getAllStandups = async () => {
...
+export const deleteStandups = async (id: string) => {
+  try {
+    const standup = await prisma.standup.delete({
+      where: {
+        id: Number(id),
+      },
+    });
+    return standup;
+  } catch (error) {
+    console.log(error);
```
Comment (SujayPrabhu96) [inline server/src/services/standups.service.ts:169]:
```suggestion
    throw error
```

**Review by SujayPrabhu96 — CHANGES_REQUESTED** [review]

File: server/src/__tests__/standup/standups.service.test.ts [hunk 4]
```diff
@@ -192,3 +193,19 @@ describe("Post Standup", () => {
     );
   });
 });
+
+// Test delete standups
+
+describe("Delete Standup", () => {
+  const mockStandups = {
```
Comment (SujayPrabhu96) [inline server/src/__tests__/standup/standups.service.test.ts:200]:
```suggestion
  const mockStandup = {
```

File: server/src/services/standups.service.ts [hunk 5]
```diff
@@ -157,6 +157,18 @@ export const getAllStandups = async () => {
   }
 };
 
+export const deleteStandups = async (id: string) => {
```
Comment (SujayPrabhu96) [inline server/src/services/standups.service.ts:160]:
```suggestion
export const deleteStandup = async (id: string) => {
```

**Review by SujayPrabhu96 — CHANGES_REQUESTED** [review]

File: server/src/services/standups.service.ts (hunk 5 above)
Comment (SyedSibtainRazvi) [reply server/src/services/standups.service.ts]: Updated

**Review by SujayPrabhu96 — APPROVED** [review]

**Pull request merged**